client/render/Canvas.h

```diff
@@ -93,8 +93,8 @@ class Canvas
 	/// renders multiple lines of text with specified parameters
 	void drawText(const Point & position, const EFonts & font, const ColorRGBA & colorDest, ETextAlignment alignment, const std::vector<std::string> & text );
 
-	/// fills selected area with solid color
-	void drawColor(const Rect & target, const ColorRGBA & color);
+	/// fills selected area with solid color, allows replacing or overdrawing
+	void drawColor(const Rect & target, const ColorRGBA & color, const bool replace = true);

```

Comment:
1) "blendColors" should be more clear than "replace"
2) Declaring method parameters as const (non-reference/pointer) is pointless since this parameter is copied into method anyway
3) OR actually - might be better to create separate method, e.g. drawColorBlended
```suggestion
	void drawColor(const Rect & target, const ColorRGBA & color)
	void drawColorBlended(const Rect & target, const ColorRGBA & color)
```